Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -634,6 +620,72 @@ namespace Multiplayer
         stats.RecordFrameTime(AZ::TimeUs{ duration.count() });
     }
 
+    void MultiplayerSystemComponent::UpdatedMetricsConnectionCount()
+    {
+        MultiplayerStats& stats = GetStats();
+        auto sendNetworkUpdates = [&stats](IConnection& connection)
```

Comment:
```suggestion
        auto updateConnectionStats = [&stats](IConnection& connection)
```
`sendNetworkUpdates` doesn't seem appropriate now that it isn't sending any updates in this lambda

Reply:
Good point, updated